packages/replayer/src/replayer.ts

```diff
@@ -148,6 +149,26 @@ export const replayEvents: ReplayEventsFn = async (options) => {
   const replayData = await initializeReplayData({ page, startUrl });
   initializeReplayDataSpan?.finish();
 
+  const loadReplayUserInteractionsExports = parentPerformanceSpan?.startChild({
+    op: "loadReplayUserInteractionsExports",
+  });
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const replayUserInteractionsExports = require(dependencies
+    .nodeUserInteractions.location);
+  loadReplayUserInteractionsExports?.finish();
+
+  const boostrapUserInteractions: BootstrapReplayUserInteractionsFn =
+    replayUserInteractionsExports.boostrapUserInteractions;
+
+  let replayUserInteractions: ReplayUserInteractionsFn;
+  if (typeof boostrapUserInteractions === "function") {
```

Comment:
Once we merge the snippet PR we can presumably get rid of this check, and just always assume it's the latest snippet?

Reply:
yep that's correct.